src/backends/dummy.rs

```diff
@@ -43,17 +43,29 @@ use std::ptr;
 /// A tracer which doesn't really do anything.
 pub struct DummyTracer {
     started: bool,
+    destroyed: bool,
```

Comment:
These two bools look like they're forming a state machine to me. Maybe there should be a single `enum` which stores the state?

Reply:
good idea.